test/e2e/rendererList.test.ts

```diff
@@ -0,0 +1,55 @@
+import * as mwoffliner from '../../src/mwoffliner.lib.js'
```

Comment:
I don't understand the purpose of this new test. It does not go through a loop to test elegantly all api-end point?!

Reply:
It tests the renderName param for bm.wikipedia.org.

Reply:
OK, I get it now